code/modules/mob/living/say.dm

```diff
@@ -88,9 +90,12 @@ proc/get_radio_key_from_channel(var/channel)
 	if(GREY in mutations)
 		message = "<span class='grey'>[message]</span>"
 
-	else if(COMIC in mutations)
+	if(COMIC in mutations)

```

Comment:
There's some condition that triggers this---as I was touching this code, I tried to replicate this by giving myself comic, slurring, stuttering, hulk, and a number of others. I then talked locally and over radio, but I couldn't trigger the telltale "<s-span-class="wa-r-rn-ning> I k-kne-w I s-shouildn't d-do that-t-t </s-spa-nn>"